modules/indexer/issues/internal/model.go

```diff
@@ -89,6 +89,8 @@ type SearchOptions struct {
 
 	MilestoneIDs []int64 // milestones the issues have
 
+	Index optional.Option[int64] // keyword as potential issue index
+
```

Comment:
It seems that it doesn't need to introduce this field.

It could use a struct method: `if opts.IsKeywordAsIssueIndex() { ... }`

Reply:
done